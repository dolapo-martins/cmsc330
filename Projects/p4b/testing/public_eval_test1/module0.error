File "evaluate.ml", line 31, characters 1-818:
Warning 8: this pattern-matching is not exhaustive.
Here is an example of a value that is not matched:
(Num _|List _|Fun (_, _, _, _)|Sum (_, _)|Greater (_, _)|Equal (_, _)|
Less (_, _)|Mult (_, _)|Pow (_, _)|Paren _)::_
