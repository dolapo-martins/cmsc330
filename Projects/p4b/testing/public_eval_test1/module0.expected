1
10
3
10
6
10
10
20
15
20
21
20
28
20
36
20
45
20

